### DISCUSSION TITLE
Webhook Options

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3105] [Feature Requests]
* [ ] Choose Included Data — Data Mapping: design JSON, inject parent/relational values
* [ ] On Interval
* [ ] Queue if > 1 request per time
* [ ] Retry on error

**Comment by emregency** [comment]
Hi @benhaynes ,

Do you think it makes sense to consider `delta` between revisions as well?

Maybe I give a bit more context. I was thinking if I store every single activity with deltas, and store them like events, I would be able to roll-back to an entire enterprise state, and do real time analytics. Does that make sense?

**Reply by benhaynes** [reply]
We could look into making anything and everything available through configuration, and then let the admin setup what they actually want to send. Activity and Revision data are relationally tied to the parent item, so that should be possible.

**Reply by emregency** [reply]
🤔a deep customization of webhooks would be cool. For now, I will go with messages and not events then. Thanks 🙏

**Comment by rijkvanzanten** [comment]
This has since been implemented through Flows!
